src/vmm/src/builder.rs

```diff
@@ -1528,14 +1478,7 @@ pub mod tests {
 
         let entropy_config = EntropyDeviceConfig::default();
 
-        let mut cmdline = default_kernel_cmdline();
-        insert_entropy_device(&mut vmm, &mut cmdline, &mut event_manager, entropy_config);
-        // Check if the vsock device is described in kernel_cmdline.
-        #[cfg(any(target_arch = "x86", target_arch = "x86_64"))]
-        assert!(cmdline_contains(
-            &cmdline,
-            "virtio_mmio.device=4K@0xd0000000:5"
-        ));
+        insert_entropy_device(&mut vmm, &mut event_manager, entropy_config);
```

Comment:
This test does not actually assert anything anymore now, so I think we can remove it altogether

Reply:
same as above